Pod/Classes/SKYAuthContainer.h

```diff
@@ -103,16 +103,16 @@ typedef void (^SKYContainerUserOperationActionCompletion)(SKYRecord *_Nullable u
 /**
  Creates a user account with the specified auth data and password.
  */
-- (void)signupWithAuthData:(NSDictionary *)authData
+- (void)signupWithAuthData:(NSDictionary<NSString *, NSString *> *)authData
```

Comment:
It should be `NSDictionary<NSString *, id>`

Reply:
Oh I didn’t know it supports something other than string : )